www/components/core/Search.tsx

```diff
@@ -93,9 +102,15 @@ export const Search = ({ descriptors }: SearchParams) => {
           {results.length > 0 &&
             results.map((result: Result, ind: number) => (
               <SearchResultItem key={`${result.item.name}_${ind}`}>
-                <ItemLink to={`${result.item.url}`}>
-                  <Match result={result} />
-                </ItemLink>
+                {useReactRouter ? (
+                  <ItemRouterLink to={`${result.item.url}`}>
+                    <Match result={result} />
+                  </ItemRouterLink>
+                ) : (
+                  <ItemLink href={`${result.item.url}`}>
+                    <Match result={result} />
+                  </ItemLink>
+                )}
```

Comment:
Should we extract this into a component? We're using this in at least 2 places now and it would be lovely to abstract away the switch between router and html links and just render the children in the correct target component.